src/routes/s3-route.js

```diff
@@ -3,6 +3,12 @@ import * as controller from '../controllers/s3-controller.js'
 
 const router = express.Router()
 
+// PUT /s3/upload - Upload object to default S3 bucket
+router.put('/upload', controller.uploadObject)
+
+// PUT /s3/upload/bulk - Upload multiple objects (original, large, thumbnail) to default S3 bucket
+router.put('/upload/bulk', controller.uploadBulkObjects)
+
```

Comment:
We can use those routes for testing, but eventually upload to S3 will be part of the media upload route